### PR TITLE
dev-lang/zig: add memory requirement

### PR DESCRIPTION
**Pull request by BratishkaErik**
See https://github.com/ziglang/zig/issues/6485

Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: BratishkaErik <bratishkaerik@getgoogleoff.me>

### CODE REVIEW

File: dev-lang/zig/zig-0.9.1.ebuild [hunk 1]
```diff
@@ -13,7 +13,7 @@ if [[ ${PV} == 9999 ]]; then
 	inherit git-r3
 else
 	SRC_URI="https://github.com/ziglang/zig/archive/${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64 ~arm ~arm64"
+	KEYWORDS="~amd64 ~arm64"
```
Comment (Flowdalic) [inline dev-lang/zig/zig-0.9.1.ebuild:16]: Why drop `~arm`?

**Comment by Flowdalic** [comment]
Could you split the commit into two commits: one "drop old" and one "add memory requirement via check-regs"?

File: dev-lang/zig/zig-0.9.1.ebuild (hunk 1 above)
Comment (BratishkaErik) [reply dev-lang/zig/zig-0.9.1.ebuild:16]: Oops, sorry...

**Comment by BratishkaErik** [comment]
> Could you split the commit into two commits: one "drop old" and one "add memory requirement via check-reqs"?

Done.

**Pull request closed without merging**